src/pages/import-cards-popup/import-cards-popup.tsx

```diff
@@ -50,26 +60,13 @@ export const ImportCardsPopup = ({ showPopup, onClose }: ImportCardsPopupProps)
           selectedButtonKey={cardSeparator}
           onButtonSelect={setCardSeparator}
         />
-        <div className="import-cards-list-item-label">3. add your cards</div>
-        <div className="import-cards-input-container">
-          <TextArea
-            lines={5}
-            placeholder="copy and paste your cards here"
-            value={textAreaValue}
-            onChange={setTextAreaValue}
-          />
-          <BubbleDivider
-            label={'or'}
-            variantColor="light"
-            className="import-cards-divider"
-            buttonClassName="import-cards-divider-button"
-          />
-          <div className="browse-files-button-container">
-            <Button variant="light" onClick={noop} className="import-cards-popup-button">
-              browse your files
-            </Button>
-          </div>
-        </div>
+        <div className="import-cards-list-item-label">3. how to import your cards?</div>
+        <RadioButtonGroup
+          radioButtonOptions={importMethodOptions}
+          selectedButtonKey={importMethod}
+          onButtonSelect={setImportMethod}
```

Comment:
do we also want to clear the `importedFile` if this changes?
(i.e. we change to the `copy and paste` and back to `upload file`)

Reply:
Yeah, I wasn't quite sure either way on this one, haha. I ended up clearing both the textbox and selected file whenever it is toggled.